android/java/res/layout/fragment_onboarding_network_selection.xml

```diff
@@ -24,10 +24,10 @@
             android:layout_marginBottom="18dp"
             android:text="@string/supported_networks"
             android:textColor="@color/wallet_text_color"
```

Comment:
Same here.

Reply:
Done in commit 👉 [`a85fb75` (#24063)](https://github.com/brave/brave-core/pull/24063/commits/a85fb75ba0e9f37da89ed68e35e9141a90eadbdf)